app/uk/gov/hmrc/disareturns/models/common/ErrorResponses.scala

```diff
@@ -98,7 +116,10 @@ object ErrorResponse {
         case "VALIDATION_FAILURE" if (json \ "errors").validate[Seq[SecondLevelValidationError]].isSuccess =>
           json.validate[SecondLevelValidationResponse]
         case "BAD_REQUEST" =>
-          badRequestErrReads.reads(json)
+          (json \ "errors").toOption match {
```

Comment:
Is `badRequestErrReads` still in use if this line has been replaced?

Reply:
Oh sorry you mean the reads on line 89 ignore me haha